pkg/cephfs/nodeserver.go

```diff
@@ -309,6 +309,80 @@ func (ns *NodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	return &csi.NodeUnstageVolumeResponse{}, nil
 }
 
+func (ns *NodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
+
+	var err error
+	targetPath := req.GetVolumePath()
+	if targetPath == "" {
+		err = fmt.Errorf("targetpath %v is empty", targetPath)
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+	volID := req.GetVolumeId()
```

Comment:
I think we need to do validation of this  volumeID as per the spec return NOTFound Error

Reply:
we need to validate the volume is mounted to this targetPath

Reply:
made it as `todo` because of unavailable `secret` option in the spec.